indra/newview/app_settings/settings.xml

```diff
@@ -7932,6 +7932,17 @@
       <key>Value</key>
       <integer>0</integer>
     </map>
+    <key>RenderDebugTextureLabel</key>
+    <map>
+      <key>Comment</key>
+      <string>Enable texture labels via glObjectLabel. Requires restart for some features.</string>
+      <key>Persist</key>
+      <integer>1</integer>
+      <key>Type</key>
```

Comment:
Make this not persist.  It will reduce performance forever when enabled, and since glObjectLabel only exists in GL 4.3 and later, enabling this on previous OpenGL versions will make the viewer crash forever on many machines if settings get saved with this enabled.  Also maybe protect calls to glObjectLabel with a GL version check.

Reply:
OK, this no longer persists. For convenience reasons, I've split the feature into two configs so that some functionality no longer requires a restart.

GL version checks are also put in place.